README.md

```diff
@@ -26,13 +30,16 @@ The Definitive Guide is written in Asciidoc and the docs repo also contains a [s
 
 The Definitive Guide is available for multiple versions of Elasticsearch:
 
-* The [branch `1.x`](https://github.com/elastic/elasticsearch-definitive-guide/tree/1.x) applies to Elasticsearch 1.x
-* The [branch `2.x`](https://github.com/elastic/elasticsearch-definitive-guide/tree/2.x) applies to Elasticsearch 2.x
-* The [branch `master`](https://github.com/elastic/elasticsearch-definitive-guide/tree/2.x) applies to master branch of Elasticsearch (the current development version)
+* The [`1.x` branch](https://github.com/elastic/elasticsearch-definitive-guide/tree/1.x) applies to Elasticsearch 1.x
+* The [`2.x` and `master` branches](https://github.com/elastic/elasticsearch-definitive-guide/tree/2.x) apply to Elasticsearch 2.x
 
 ## Contributing
 
-Before contributing a change please read our [contribution guide](CONTRIBUTING.md).
+This repository is no longer maintained. Pull requests and issues will not be
+addressed.
+
+To contribute to the current Elasticsearch docs, refer to the [Elasticsearch
+repo](https://github.com/elastic/elasticsearch/).
```

Comment:
Very minor nitpick: Should we spell out "repo" for consistency?

Reply:
Yes. Thanks!